src/Components/Artwork/FlatGridItem.tsx

```diff
@@ -133,7 +131,6 @@ export const FlatGridItemFragmentContainer = createFragmentContainer(
           @arguments(
             includeConsignmentSubmission: $includeConsignmentSubmission
           )
-        ...DeprecatedSaveButton_artwork
```

Comment:
Removing the last of these personalized 'save' bits from the overall `Artwork` query.